.azure-pipelines.yaml

```diff
@@ -21,7 +21,10 @@ jobs:
   - bash: echo "##vso[task.prependpath]$CONDA/bin"
     displayName: Add conda to PATH.
 
-  - bash: conda update conda --yes --quiet
```

Comment:
Why is that necessary? @tobiasraabe!

Reply:
@MaxBlesch : https://github.com/conda/conda/issues/9337

Reply:
Ok thanks!